embedded-ebpf.go

```diff
@@ -0,0 +1,12 @@
+//go:build ebpf
+// +build ebpf
+
+package tracee
```

Comment:
Why not having a pkg "embed" instead?

Reply:
```
tracee-ebpf/main.go:19:2: found packages embed (embedded-ebpf.go) and tracee (embedded.go) in /home/rafaeldtinoco/work/sources/ebpf/tracee
```

This is a change being coordinated with @danielpacak and its part of the tree reorganization.

Reply:
`embedded*.go` files are supposed to be in the root directory of the project because Go embed package does not support relative paths such as `../dist/some-file.o`. On the other hand, we named our single Go module, which has its go.mod in the root dir, `github.com/aquasecurity/tracee`. That's why I thought that importing the main package and giving it `embed` or `embedded` alias will do the job.